### DISCUSSION TITLE
Welcome to Clusternet Discussions!

### DISCUSSION THREAD

**Discussion by dixudx** [clusternet/clusternet#427] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ayushgml** [comment]
Hey everyone! I am Ayush and I am a student who is learning about cloud native. Moreover I write blogs and I have also contributed to a couple of CNCF projects. Although I am thinking of contributing to Clusternet too, I am unable to find good first issues to get started with. I would like to ask maintainers and other community members if there is any issue appropriate for a first timer?

**Reply by dixudx** [reply]
@ayushgml Welcome to join us. I think #729 is a good start. This will help you know more about the code base.

**Reply by ayushgml** [reply]
Thank you @dixudx! Yeah, I will start working on the issue.
